internal/config/timeouts.go

```diff
@@ -0,0 +1,31 @@
+package config
```

Comment:
I would try to get rid of this package and just parse the timeouts option when loading the configuration. Also not make it part of the provisioner but instead adding it to the `Provider` type which already holds configurations for the cloud platform being targeted.

Reply:
This is a helper function that is used during the `loadConfigurations` it is extracted to the separate package to not to duplicate logic. https://github.com/kyma-incubator/hydroform/pull/53/files#diff-dfa71d4200e54a500c09fed237cbbecdR230
I am open to changing the place of this function but I am not sure if it should be the configuration in the `Provider` as it is not really related to the provider but rather to underlying mechanisms of Hydroform (Terraform in this case).

Reply:
The not duplicating makes sense definitely yes, and we might achieve it by pushing some of the parsing logic into the terraform operator. 
But the main goal of Hydroform is to abstract away the operator (terraform in this case).

Unfortunately the current implementation breaks the abstraction by adding those timeouts to the provisioners. We need to find a way to add this without leaking terraform specifics into hydroform.

Maybe we can move the load configuration common logic into the terraform operator and pass those timeouts as option to the API calls. That way the operator would be able to add those map entries if needed without disturbing the provisioners.